packages/strapi-parts/src/MainNav/NavLink.js

```diff
@@ -57,8 +58,27 @@ const MainNavRow = styled(Row)`
   padding: ${({ theme }) => `${theme.spaces[2]} ${theme.spaces[3]}`};
 `;
 
-export const NavLink = ({ children, icon, ...props }) => {
+const Notification = styled(Row)`
+  //positionning when condensed
```

Comment:
You can group css prop depending on a specific state by using a function returning a template string

```
  ${({ condensed }) => `
    position: absolute;
    transform: translate(35%, -50%);
    ...
`}
```